src/handlers/manaToken.ts

```diff
@@ -57,7 +68,12 @@ export function handleMint(event: Mint): void {
       accountTo = new Account(accountToId)
     }
 
-    accountTo.mana = accountTo.mana.plus(event.params.amount)
+    let mana = accountTo.mana
+    if (mana !== null) {
```

Comment:
I think you should use != here. Can you check how we are doing it on the other subgraphs?

Reply:
updated! 🙌